packages/@aws-cdk/service-spec-importers/src/full-database.ts

```diff
@@ -0,0 +1,34 @@
+import path from 'node:path';
+import { SpecDatabase, emptyDatabase } from '@aws-cdk/service-spec-types';
+import { DatabaseBuilder, DatabaseBuilderOptions } from './db-builder';
+
+const SOURCES = path.join(__dirname, '../../../../sources');
```

Comment:
This makes me think this file needs to live somewhere else. Can it be an unpackaged build tool for `aws-service-spec` ?

Otherwise, make this directory a parameter I think and call it with the right relative directory from the `aws-service-spec` build.

Reply:
Yes planning to do exactly this in the next PR!